tests/to92l.test.ts

```diff
@@ -7,3 +7,9 @@ test("to92l", () => {
   const svgContent = convertCircuitJsonToPcbSvg(circuitJson)
   expect(svgContent).toMatchSvgSnapshot(import.meta.path, "to92l")
 })
+
+test("to92l_inline", () => {
+  const circuitJson = fp.string("to92l_inline").circuitJson()
+  const svgContent = convertCircuitJsonToPcbSvg(circuitJson)
+  expect(svgContent).toMatchSvgSnapshot(import.meta.path, "to92l_inline")
```

Comment:
there is a bug in this test snapshot, maybe due to an old version of circuit-json and/or circuit-to-svg, @techmannih can help resolve

Reply:
(the pills don't appear)